oid4vc/pyproject.toml

```diff
@@ -32,10 +33,12 @@ cbor2 = {version = "~5", optional = true}
 cbor-diag = {version = "*", optional = true}
 cwt = {version = "~2", optional = true}
 pycose = {version = "~1", optional = true}
+jsonpointer = "^3.0.0"
```

Comment:
This would need to be marked as optional for it to be loadable as a separate extra